README.md

```diff
@@ -22,8 +22,11 @@ In the diagram above:
 
 ## Test Configuration
 
-The Test Network Function support auto-configuration using labels and annotations, but also a static configuration using a file. The following sections describe how to configure the TNF via labels/annotation and the corresponding settings in the config file. A sample config file can be found [here](test-network-function/tnf_config.yml).
+The Test Network Function support auto-configuration using labels and annotations. The following sections describe how to configure the TNF via labels/annotation and the corresponding settings in the config file. A sample config file can be found [here](test-network-function/tnf_config.yml).
 
+### targetNameSpaces
+
+A single namespace should be specified in the [configuration file](test-network-function/tnf_config.yml). This namespace will be used by auto-configuration to discover the Pods under test. It's preferred not to run multiple tests in different namespaces in parallel, some tests are intrusive and can cause failures if two tests are running simultaneously.
```

Comment:
to be consistent, autodiscovery instead of auto-configuration

Reply:
updated. please check new formulation